pennylane/ops/functions/iterative_qpe.py

```diff
@@ -84,7 +76,7 @@ def circuit():
                                                                                                                       ╚═══════╡ ╰Sample[MCM]
     """
     missing = []
-    if aux_wire == "unset" and ancilla == "unset":
+    if aux_wire == "unset":
         missing.append("'aux_wire'")
     if iters == "unset":
         missing.append("'iters'")
```

Comment:
You can remove this whole section once the arguments are changed to positional arguments.